examples/tv-casting-app/linux/main.cpp

```diff
@@ -114,6 +115,31 @@ HelpOptions helpOptions("tv-casting-app", "Usage: tv-casting-app [options]", "1.
 
 OptionSet * allOptions[] = { &cmdLineOptions, &helpOptions, nullptr };
 
+static void OnBindingChanged(const EmberBindingTableEntry & binding)
+{
+    if (binding.type == EMBER_MULTICAST_BINDING)
+    {
+        ChipLogError(NotSpecified,
+                     "Group binding received nodeId=0x" ChipLogFormatX64 " remote endpoint=%d cluster=" ChipLogFormatMEI,
+                     ChipLogValueX64(binding.nodeId), binding.remote, ChipLogValueMEI(binding.clusterId.ValueOr(0)));
```

Comment:
binding.nodeId is not valid for a multicast binding.  Neither is binding.remote.  This is a binding to a _group_, not a (node, endpoint) pair.

Reply:
good catch, fixed